tests/contrib/gp/test_models.py

```diff
@@ -60,7 +71,23 @@
 
 
 @pytest.mark.parametrize("model_class, X, y, kernel, likelihood", TEST_CASES, ids=TEST_IDS)
-def test_model_forward(model_class, X, y, kernel, likelihood):
+def test_model(model_class, X, y, kernel, likelihood):
+    if model_class is SparseGPRegression or model_class is SparseVariationalGP:
+        gp = model_class(X, None, kernel, X, likelihood)
+    else:
+        gp = model_class(X, None, kernel, likelihood)
+
+    loc, var = gp.model()
+    if model_class is VariationalGP or model_class is SparseVariationalGP:
+        assert_equal(loc.norm().item(), 0)
+        assert_equal(var, torch.ones(var.shape[-1]).expand(var.shape))
+    else:
+        assert_equal(loc.norm().item(), 0)
+        assert_equal(var, kernel(X).diag())
```

Comment:
Here are model tests. `loc`, `var` of variational model come from variational parameters.